SCAFFOLDING.md

```diff
@@ -14,7 +14,7 @@ Before writing a `webpack-cli` scaffold, think about what you're trying to achie
 
 ## webpack-addons-yourpackage
 
-In order for `webpack-cli` to compile your package, it relies on a prefix of `webpack-addons`. The package must also be published on npm. If you are curious about how you can create your very own `addon`, please read [How do I compose a webpack-addon?](https://github.com/ev1stensberg/webpack-addons-demo).
+In order for `webpack-cli` to compile your package, it relies on a prefix of `webpack-addons`. If you are curious about how you can create your very own `addon`, please read [How do I compose a webpack-addon?](https://github.com/ev1stensberg/webpack-addons-demo).
```

Comment:
Change this to describe diff from local packages ( no prefix needed ) and npm ones

Reply:
Ah OK, I didn't realize that local packages didn't have to have the prefix.